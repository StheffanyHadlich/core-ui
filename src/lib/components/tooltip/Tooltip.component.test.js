import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/tooltip/__snapshots__"
});
