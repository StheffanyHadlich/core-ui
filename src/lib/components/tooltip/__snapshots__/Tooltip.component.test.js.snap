// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Tooltip Default 1`] = `
<div>
  <h3>
    Right (Default) 
  </h3>
  <div
    class="tooltipDiv"
  >
    <div
      className="sc-tooltip sc-bdVaJa bDWFJH"
    >
      <span
        class="tooltipText"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        Hover me!
      </span>
    </div>
  </div>
  <h3>
    Top
  </h3>
  <div
    class="tooltipDiv"
  >
    <div
      className="sc-tooltip sc-bdVaJa bDWFJH"
    >
      <span
        class="tooltipText"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        Hover me!
      </span>
    </div>
  </div>
  <h3>
    Bottom
  </h3>
  <div
    class="tooltipDiv"
  >
    <div
      className="sc-tooltip sc-bdVaJa bDWFJH"
    >
      <span
        class="tooltipText"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        Hover me!
      </span>
    </div>
  </div>
  <h3>
    Left
  </h3>
  <div
    class="tooltipDiv"
  >
    <div
      className="sc-tooltip sc-bdVaJa bDWFJH"
    >
      <span
        class="tooltipText"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        Hover me!
      </span>
    </div>
  </div>
</div>
`;
